src/Makefile.am

```diff
@@ -815,10 +815,11 @@ check_PROGRAMS += tests/testClpMap
 tests_testClpMap_SOURCES = \
 	tests/testClpMap.cc
 nodist_tests_testClpMap_SOURCES = \
+	base/ClpMap.h \
```

Comment:
Why do we want to list this particular header file in `nodist_tests_testClpMap_SOURCES`? It was not listed before this PR, and other header files (included by testClpMap.cc) remain unlisted in this PR, so this looks like an out of scope (and possibly unnecessary) change, but perhaps I am missing something.

Reply:
I think this failed some build until I did, but now after retesting it seems to work. Removing.